components/Tag/index.d.ts

```diff
@@ -3,6 +3,7 @@ import React from 'react';
 export interface TagProps {
     bold?: boolean;
     inverted?: boolean;
+    children?: React.ReactNode;
```

Comment:
Changes this if it needs to use multiple children.

```suggestion
    children?: React.ReactNode | React.ReactNode[];
```